design/schedule-unpaused-triggers-config_design.md

```diff
@@ -0,0 +1,102 @@
+# Schedule Unpaused Triggers Config Design
+## Abstract
+User want to configure what happens when schedule is unpaused. Either run backup immediately or at the next cron schedule.  This design proposes to add a new field to the schedule spec to allow user to configure the behavior
+
+## Background
+Currently, the default behavior is to run the trigger immediately and may not be desired by all users (https://github.com/vmware-tanzu/velero/issues/6517)
+
+## Goals
+- Make schedule unpaused behavior configurable
+
+## Non Goals
+- Changing the default behavior
+
+
+## High-Level Design
+Add a new field with to the schedule spec to allow user to configure the behavior defaulting to prior behavior of triggering backup immediately.
+
+## Detailed Design
+### CLI Changes
+`velero schedule unpause` will now take an optional bool flag `--unpause-triggers` to allow user to override the behavior configured in the schedule spec.
+
+`velero schedule unpause schedule-1 --unpause-triggers=false` will unpause the schedule but not trigger the backup immediately.
+
+`velero schedule unpause schedule-1 --unpause-triggers=true` will unpause the schedule and trigger the backup immediately.
+
+`velero schedule unpause schedule-1` will check `.spec.UnpauseTriggers` in the schedule to determine behavior. This field will default to true to maintain prior behavior.
+
+### API Changes
+`pkg/apis/velero/v1/schedule_types.go`
+```diff
+// ScheduleSpec defines the specification for a Velero schedule
+type ScheduleSpec struct {
+	// Template is the definition of the Backup to be run
+	// on the provided schedule
+	Template BackupSpec `json:"template"`
+
+	// Schedule is a Cron expression defining when to run
+	// the Backup.
+	Schedule string `json:"schedule"`
+
+	// UseOwnerReferencesBackup specifies whether to use
+	// OwnerReferences on backups created by this Schedule.
+	// +optional
+	// +nullable
+	UseOwnerReferencesInBackup *bool `json:"useOwnerReferencesInBackup,omitempty"`
+
+	// Paused specifies whether the schedule is paused or not
+	// +optional
+	Paused bool `json:"paused,omitempty"`
+
++	// UnpauseTriggers specifies whether to trigger backup immediately when schedule is unpaused
```

Comment:
Also, add that the default value is true if not set by the user, this comment gets converted to CRD descriptions so it will be helpful for the users.